### PR TITLE
[DOC release] document current-when argument for link-to #12038

### PR DESCRIPTION
**Pull request by EricSchank**
Add docs for `current-when` on `link-to`

See issue #12038

### CODE REVIEW

File: packages/ember-routing-htmlbars/lib/keywords/link-to.js [hunk 1]
```diff
@@ -139,7 +139,35 @@ import merge from 'ember-metal/merge';
...
+  argument.
+  
+  ```handlebars
+  {{#link-to 'photoGallery' current-when='photos'}}
+    Photo Gallery
+  {{/link-to}}
+  ```
+  
+  This may be helpful for keeping links active for:
+  * non-nested routes that are logically related
```
Comment (mixonic) [inline packages/ember-routing-htmlbars/lib/keywords/link-to.js]: Does there need to be a newline before the list for it to be a list? Did you preview this locally?

Comment (btecu) [inline packages/ember-routing-htmlbars/lib/keywords/link-to.js]: Doesn't `current-when` have to be in quotes because of the dash?

Comment (mixonic) [inline packages/ember-routing-htmlbars/lib/keywords/link-to.js]: @btecu can you expand on that? Give an example?

Comment (btecu) [inline packages/ember-routing-htmlbars/lib/keywords/link-to.js]: Variable names that contain a dash need to be in quotes. Is that not the case here?

**Comment by EricSchank** [comment]
@mixonic 
Yep, missed the blank line while editing on GH.
Verified locally that list looks right now.
Thanks!

**Comment by rwjblue** [comment]
Awesome, thank you!

**Pull request merged**